tests/unit/doc-printer.js

```diff
@@ -40,6 +40,31 @@ test("`printDocToString` should not manipulate docs", () => {
 
     const lines = formatted.split("\n");
     expect(lines.length).toBeGreaterThan(1000);
-    expect(endTime - startTime).toBeLessThan(1000);
+    expect(endTime - startTime).toBeLessThan(100);
```

Comment:
On my machine, it was:
- 181.9 on origin/main
- 13.3 on this branch

It get x10~ faster so dividing by 10 should be safe.

Reply:
Repeating the test, 181.9 on origin/main looks to be a bad case.
It can be sometimes done in about 100ms.